code/__DEFINES/dcs/signals.dm

```diff
@@ -67,6 +67,9 @@
 #define COMSIG_GLOB_CAS_LASER_CREATED "!cas_laser_sent"
 #define COMSIG_GLOB_RAILGUN_LASER_CREATED "!railgun_laser_sent"
 
+//Signals for fire support
```

Comment:
bruh what is this copy past comment? Also make them (both) autodoc, not a comment